recipe/meta.yaml

```diff
@@ -24,7 +23,7 @@ requirements:
     - hatch-fancy-pypi-readme
   run:
     - python >=3.8
-    - awkward-cpp ==38
+    - awkward-cpp ==39
```

Comment:
Basically by the nature of how Awkward works, bumping the `awkward-cpp` version is going to be required a lot. c.f. https://github.com/scikit-hep/awkward/commit/283fa618c02f2e8406609684c07f8a9512df169d